test/Prototypes/NoPrototypeForMethod.js

```diff
@@ -0,0 +1,15 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var o = {
+    f() { }
+}
+WScript.Echo(o.f.hasOwnProperty("prototype"));

```

Comment:
instead of creating a baseline can we have logic in the test itself to validate what would be result we are expecting? if any validation failed we could print 'Failed' otherwise 'Pass'?